www/locales/en.json

```diff
@@ -195,7 +195,7 @@
     "WRITER": "Writer",
     "SOURCES": {
       "ALL_SOURCES": {
-        "TEXT": "All Sources"
+        "TEXT": "All sources"
```

Comment:
i feel like the second word being capitalized improved readability

Reply:
@tarunsingh5 veerji, yeah I reverted this change back, this was a sideeffect of another text change we did earlier but wasn't pushed.